unittests/finality_proof.hpp

```diff
@@ -168,21 +172,26 @@ namespace finality_proof {
 
          //skip this part on genesis
          if (!is_genesis){
+            parent_timestamp = timestamp;
             for (const auto& p : blocks_since_proposed_policy){
-               //under the happy path with strong QCs in every block, a policy becomes active `2 * eosio::testing::num_chains_to_final` blocks after being proposed
-               if (p.second.blocks_since_proposed == 2 * eosio::testing::num_chains_to_final){
+
+               //under the happy path with strong QCs in every block, a policy becomes active 6 blocks after being proposed

```

Comment:
If we don't use the constant `num_chains_to_final` in the comment, 6 should be 4

Reply:
Done